_posts/en/newsletters/2021-02-10-newsletter.md

```diff
@@ -0,0 +1,231 @@
+---
+title: 'Bitcoin Optech Newsletter #135'
+permalink: /en/newsletters/2021/02/10/
+name: 2021-02-10-newsletter
+slug: 2021-02-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to the summary of last week's taproot
+activation meeting and announces another scheduled meeting for next
+week, plus describes recent progress in discreet log contracts and a new
+mailing list for discussing them.  Also included are our regular
+sections with the summary of a Bitcoin Core PR Review Club meeting,
+descriptions of releases and release candidates, and a list of notable
+changes to popular Bitcoin infrastructure projects.
+
+## News
+
+- **Taproot activation meeting summary and follow-up:** Michael Folkson
+  posted a [summary][folkson1] of the scheduled meeting to discuss taproot
+  activation methods (see [Newsletter #133][news133 taproot meeting]).
+  There appeared to be support among participants for using the [BIP8][]
+  activation mechanism with the earliest allowed activation time about
+  two months after the first release of Bitcoin Core containing the
+  activation code and the latest allowed activation of that initial
+  deployment about one year subsequently.
+
+    More controversial was whether the `LockinOnTimeout` (LOT) parameter
+    should be set to *true* (requiring miners either eventually signal
```

Comment:
not sure it is wrong as-is but for similar construction as the next clause ("allowing miners to signal...")
```suggestion
    should be set to *true* (requiring miners to either eventually signal
```